src/components/pieChart/pieChart.tsx

```diff
@@ -72,14 +73,10 @@ class PieChart extends React.Component<PieChartProps, State> {
               labels={this.getTooltipLabel}
               height={height}
               width={width}
+              // do not change chart's size on resize.
+              containerComponent={<VictoryContainer responsive={false} />}
```

Comment:
Can you use ChartContainer from PF4? Should just be a wrapper around VictoryContainer.